core/src/main/java/org/apache/iceberg/deletes/Deletes.java

```diff
@@ -125,6 +126,25 @@ public static StructLikeSet toEqualitySet(
     }
   }
 
+  public static <T extends StructLike> CharSequenceMap<PositionDeleteIndex> toPositionIndexes(
```

Comment:
Unlike `toPositionIndex` below, this one does not filter the deletes for a particular data file. Instead, it builds an index for each referenced data file and returns a map.

Reply:
Added.